app/views/homepages/index.html.erb

```diff
@@ -0,0 +1,65 @@
+
+<main>
+      <section class="spotlight">
+    <h2 class="spotlight__header">
+      <span class="spotlight__header--prefix">Media Spotlight</span>
+      <%= link_to Work.spotlight.title, work_path(Work.spotlight.id), class: "spotlight__link-to" %> by <%= Work.spotlight.creator%>
+    </h2>
+    <p class="spotlight__description"><%=Work.spotlight.votes.count%> votes - <%=Work.spotlight.description%>
+    </p>
+  </section>
+
+<hr class="root__hr">
+
+<section class="top-ten__container">
```

Comment:
This is a great place for a partial view since you essentially repeat the same code 3 times.